### ISSUE TITLE
running artisan lingua:generate on Windows seems to create a directory issue?

### ISSUE THREAD

**Issue by benwrigley** [cyberwolf-studio/lingua#1]
Firstly, thank you so much for writing a brilliant package. I'm loving using it in my Vue components!

I highly suspect this is me doing something wrong, but I'm still very new to Laravel.

I'm using Homestead on a Windows machine and while my sites are being served from within Homestead I am running my Vite server on Windows but I can't get it to auto-generate the language files.

Then I found that when I run `php artisan lingua:generate` in the Windows environment I get this error:

```
Symfony\Component\Finder\Exception\DirectoryNotFoundException 

  The "G:\Sites\Laravel\everyoneinmind\lang\G:\Sites\Laravel\everyoneinmind\lang\en" directory does not exist.

  at vendor\symfony\finder\Finder.php:640
    636▕             } elseif ($glob = glob($dir, (\defined('GLOB_BRACE') ? \GLOB_BRACE : 0) | \GLOB_ONLYDIR | \GLOB_NOSORT)) {
    637▕                 sort($glob);
    638▕                 $resolvedDirs[] = array_map($this->normalizeDir(...), $glob);
    639▕             } else {
  ➜ 640▕                 throw new DirectoryNotFoundException(sprintf('The "%s" directory does not exist.', $dir));
    641▕             }
    642▕         }
    643▕
    644▕         $this->dirs = array_merge($this->dirs, ...$resolvedDirs);

  1   vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php:599
      Symfony\Component\Finder\Finder::in("G:\Sites\Laravel\everyoneinmind\lang\G:\Sites\Laravel\everyoneinmind\lang\en")

  2   vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php:353
      Illuminate\Filesystem\Filesystem::allFiles("G:\Sites\Laravel\everyoneinmind\lang\G:\Sites\Laravel\everyoneinmind\lang\en")
```

It looks like the directories have been concatenated together but I'm not sure where this happens or if it's a configuration I have messed up somewhere.

Running the same command inside Homestead runs fine, so I am manually generating for now.

**Comment by mikield** [comment]
Hello :)

Have no Windows machine to test that, but it seems that it could be related with homestead.
Maybe some folder mapping issues.

The error comes from `symfony/finder` that Laravel uses to work with files on filesystem.
Package uses `File` facade to get all dirs based on Laravel's `lang_path` helper.

**Comment by benwrigley** [comment]
Thanks for coming back so quickly! :)

Forgive me but I'm not sure how best to show this but I've narrowed it down to how the path is replaced in the `CommandTranslationGenerator::generate` method

```
        foreach ($directories as $dir) {
            $path = str_replace(lang_path() . '/', '', $dir);
            $locales[] = $path;
        }
```

I think that because Windows paths use backslashes instead. I have changed this on mine now to be

```
        foreach ($directories as $dir) {
            $path = str_replace(lang_path() . DIRECTORY_SEPARATOR, '', $dir);
            $locales[] = $path;
        }
```

which seems to run fine in both environments :)

**Comment by mikield** [comment]
Cool, so that was indeed error by my side.

You are welcome to push a PR :)

**Comment by benwrigley** [comment]
Cool! Ashamed to say I've not done a PR before so I hope I've got it right! :)

**Comment by mikield** [comment]
Thank you for your contribution, have released a new tag `v0.9.3` that contains your change.
